docs(passport.md): explains express authInfo populate procedure (#3033)
nestjs/docs.nestjs.com: content/recipes/passport.md

As a result of all this, our response to the `validate()` callback is trivial: we simply return an object containing the `userId` and `username` properties. Recall again that Passport will build a `user` object based on the return value of our `validate()` method, and attach it as a property on the `Request` object.

> Additionally, you can return an array, which first value will be used to build a `user` object and second value will be used to build a `authInfo` object.

Reviewer: ```suggestion
Additionally, you can return an array, where the first value is used to create a `user` object and the second value is used to create an `authInfo` object.
```
